### ISSUE TITLE
Order of output when producing complete book in RStudio (gitbook, pdf_book, epub_book)

### ISSUE THREAD

**Issue by philomonk** [rstudio/bookdown#210]
I run bookdown in the RStudio app version 0.99.1290. I've noticed that during building the book in all formats at once it creates the output in the following order: html, pdf, epub:

![bildschirmfoto 2016-09-23 um 11 24 12](https://cloud.githubusercontent.com/assets/12614453/18782908/b4c97bb2-8180-11e6-904e-e0e8f650365a.png)

I'm sure there are reasons for building the output in this order that I'm not aware of. One effect that that has though is that during the first build of the project the pdf and epub output don't exist yet at the time when the html is being built. As a result, the html output does not contain the menu item or links to downloading the pdf or epub:

![bildschirmfoto 2016-09-23 um 11 23 44](https://cloud.githubusercontent.com/assets/12614453/18783127/07650cd2-8182-11e6-8f39-7fa32ce37dbb.png)

Upon any subsequent build, the link shows up, I imagine, since there are a pdf and epub present (if from the previous build) at the time of building the html.

**Comment by yihui** [comment]
What is the order of these formats you specified in `_output.yml` (or YAML of index.Rmd)?

**Comment by philomonk** [comment]
The order was gitbook>pdf_book>epub_book as per bookdown-demo `_output.yml`.

You're right, RStudio takes its order of compiling output formats from the `_output.yml`. I changed the order to pdf_book>epub_book>gitbook and the problem disappears. The html output shows the download menu item and respective links to pdf and epub on first build now.

I've created a pull request with the changed order of output formats in the `_output.yml` of bookdown-demo.
